src/extension/importer/core/src/test/java/org/geoserver/importer/ImporterDataTest.java

```diff
@@ -431,6 +435,10 @@ public void testImportIntoDatabase() throws Exception {
 
         runChecks("archsites");
         runChecks("bugsites");
+        
+        // Test that temporary data is removed also
+        context.delete();
```

Comment:
If you're calling delete manually, you're testing the delete cleans up, but not if the delete will be called by the importer machinery automatically. Probably best to add a REST test and do the checks there too, without a manual delete call?
The JobQueue cleanup frequency should be made configurable, and I guess more frequent than "one minute" per loop (or the test will be too long).